test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
I'm not sure if it would actually cause problems, but the mantissa wouldn't be the same because the exponent of floats represents the binary exponent, so you only get lossless multiplication by powers of 2.